start.py

```diff
@@ -679,6 +703,36 @@ def _configure_kudu(deployment, cluster, kudu_version):
             # Only one parcel can be activated at a time, so once one is found, our work is done.
             break
 
+
+def _validate_kudu_version(kudu_version, cdh_version):
+    cdh_version_tuple = tuple([int(i) for i in cdh_version.split('.')])
+    cdh_5_13_0_tuple = 5, 13, 0
+
+    cdh_version_from_map = KUDU_PARCEL_VERSIONS.get(kudu_version, None)
```

Comment:
`.get()` defaults to `None` if the element isn't in the dictionary.

Reply:
Done.